python/cudf/cudf/tests/test_series.py

```diff
@@ -1584,6 +1584,50 @@ def test_isin_numeric(data, values):
     assert_eq(got, expected)
 
 
+@pytest.mark.xfail(raises=ValueError)
+def test_fill_new_category():
+    gs = cudf.Series(pd.Categorical(["a", "b", "c"]))
+    gs[0:1] = "d"
+
+
+@pytest.mark.parametrize("replace", [True, False])
+def test_sample(
```

Comment:
This test looks very very similar to the DataFrame version. I'm wondering if maybe there's a way to just parametrize the type and otherwise reuse the rest of the test code for the two classes.

Reply:
In fact, I'm tempting to remove it because it's testing the same code path for `axis=1` use case. But I could be wrong because we might want to change the underlying implementation in the future but we still want to make sure the API is tested.

Reply:
Yeah I wouldn't remove it since we may eventually change the impl for one of the classes for some reason.